src/main/java/com/synopsys/integration/detect/tool/detector/DetectExecutableRunner.java

```diff
@@ -56,6 +58,15 @@ public static DetectExecutableRunner newInfo(EventSystem eventSystem) {
     public ExecutableOutput execute(final Executable executable) throws ExecutableRunnerException {
         final ExecutableOutput output = super.execute(executable);
         eventSystem.publishEvent(Event.Executable, output);
+        if (output.getReturnCode() != 0) {
+            if (StringUtils.isNotBlank(output.getStandardOutput())) {
```

Comment:
Can we add a log before each output indicating whether or not the output is standard or error?